src/App/Fossa/Container.hs

```diff
@@ -206,3 +220,41 @@ syftCommand bin (ImageText image) =
       cmdArgs = ["-o", "json", image],
       cmdAllowErr = Never
     }
+
+parseSyftOutputMain :: Severity -> FilePath -> IO ()
+parseSyftOutputMain logseverity path = withLogger logseverity . logWithExit_ . runReadFSIO $ parseSyftOutput path
+
+parseSyftOutput :: (Has Diagnostics sig m, Has ReadFS sig m, Has (Lift IO) sig m, Has Logger sig m) => FilePath -> m ()
+parseSyftOutput filepath = do
+  curdir <- sendIO getCurrentDir
+  logDebug "Resolving file"
+  path <- resolveFile curdir $ pack filepath
+  logDebug "Reading JSON file"
+  rawvalue <- readContentsJson @Value path
+  logDebug "Parsing JSON contents"
+  response <- fromEitherShow $ parseEither (parseJSON @SyftResponse) rawvalue
+  logDebug "Converting to FOSSA payload"
+  payload <- toContainerScan response
+  logInfo "Payload is valid!"
+
+  sendIO . BL.putStr $ encode payload
```

Comment:
It doesn't matter much here, but we do have `logStdout` for this

Reply:
I got stuck trying to do this and forgot to fix it.  Lazy `Bytestring` doesn't have a `Pretty` instance, and I couldn't find `Data.Text.Lazy.Encoding (decodeUtf8)` until just now.

Good callout, btw.  We should actually be much stricter about this because the write ordering is non-deterministic when using the logger mixed with raw IO.  Because the logging effects are sequenced, using the logger is deterministic.

Reply:
Given that these are printed to different handles, it doesn't make a difference programmatically, but in a shell with no redirection, there's no guarantee they won't clobber each other.